pkg/sql/randgen/mutator.go

```diff
@@ -832,6 +833,8 @@ func postgresCreateTableMutator(
 				}
 			}
 			stmt.Defs = newdefs
+		case *tree.SetVar:
+
```

Comment:
hmm, does this mean we lose out on stuff like `SET intervalstyle = '...'`, and do we (deliberately) want to do that?